x-pack/plugins/ml/public/application/services/ml_api_service/index.ts

```diff
@@ -119,6 +125,13 @@ export const ml = {
     });
   },
 
+  forceCloseJob({ jobId }: { jobId: string }) {
```

Comment:
There is already a `closeJob` method. You should probably just extend it with a new param.

Reply:
A standalone function was added to be consistent with the other "force" functions in this file.